src/Elliptic/Systems/Elasticity/FirstOrderSystem.hpp

```diff
@@ -33,64 +30,33 @@ struct Operand;
 namespace Elasticity {
 
 /*!
- * \brief The Elasticity equation formulated as a set of coupled first-order
- * PDEs.
+ * \brief The linear elasticity equation formulated as a set of coupled
+ * first-order PDEs.
  *
- * \details This system formulates the Elasticity equation \f$\nabla T(x) =
- * f(x)\f$ on a euclidean space as the set of coupled first-order PDEs
+ * This system formulates the Elasticity equation \f$\nabla_i T^{ij}(x) =
+ * f^j(x)\f$. It introduces the symmetrical strain tensor S as an auxialiary
```

Comment:
Please write `f_\mathrm{ext}^j` and link to the documentation of the `Elasticity` namespace by just writing "(see \`Elasticity\`)".
That documentation is in `Elasticity/Tags.hpp`. Please make sure it is descriptive enough and adjust if you find it needs more information.  In particular, it should make clear that f_ext are the "external" forces acting on the material (counteracted by the "internal" forces -div(T) that are defined in Blandford&Thorne Eq. 11.13). Please also fix `u`->`\xi` while you're editing the docs.